mars/core.py

```diff
@@ -27,6 +27,8 @@
 from .tiles import Tilesable, handler
 from .graph import DAG
 
+_sorted_slots = dict()
```

Comment:
Maybe treating it as a class property of `Base` is a better solution, it looks a bit weird here.

Reply:
Use __key_{cls.__name} to store slots for every class.